prog/vm/host_nexus.rb

```diff
@@ -183,6 +191,25 @@ def self.assemble(sshable_hostname, location: "hetzner-hel1", net6: nil, ndp_nee
     nap 30
   end
 
+  label def destroy
+    decr_destroy
+
+    unless vm_host.allocation_state == "draining"
+      vm_host.update(allocation_state: "draining")
+      nap 5
```

Comment:
No need to nap here, right? We can proceed with next check.

Reply:
Since allocation can occur simultaneously with the updating of the ﻿allocation_state bit, it's advisable to wait 5 seconds to ensure that no virtual machine is allocated to this host during the process.